src/plugins/template/tests/functional/op_reference/group_convolution_backprop.cpp

```diff
@@ -487,6 +487,18 @@ std::vector<GroupConvolutionBackpropDataOutShapeParams> generateGroupConvolution
                                                    {1, 1},
                                                    {2},
                                                    {1, 14}),
+        // Adding new test case when output shape is non empty
+        GroupConvolutionBackpropDataOutShapeParams(Shape{1, 2, 2, 2},
+                                                   Shape{2, 1, 1, 2, 2},
+                                                   Shape{1, 2, 3, 3},
+                                                   IN_ET,
+                                                   std::vector<T>{1, 2, 3, 4, 5, 6, 7, 8},
+                                                   std::vector<T>{1, 0, 0, 1, 1, 0, 0, 1},
+                                                   std::vector<T>{1, 2, 3, 4, 5, 6, 7, 8, 9},
+                                                   {1, 1},
+                                                   {1, 1},
+                                                   {2},
+                                                   {1, 3}),
```

Comment:
Issue #24254 refers GPU plugin, not Template. Changes should be applied into [src/plugins/intel_gpu/tests/functional/shared_tests_instances/single_layer_tests/group_convolution_backprop_data.cpp](https://github.com/openvinotoolkit/openvino/blob/000c8b73cf62818d7cf340f6e91988c350b278fe/src/plugins/intel_gpu/tests/functional/shared_tests_instances/single_layer_tests/group_convolution_backprop_data.cpp#L116), not here.

Reply:
Moreover this test fails in the CI - before pushing new changes these tests should pass:
./bin/intel64/Release/ov_gpu_func_tests
./bin/intel64/Release/ov_gpu_unit_tests